content/help/system-requirements.adoc

```diff
@@ -82,25 +82,61 @@ The software prerequisites for installing on a macOS system are as follows:
 [%hardbreaks]
 === GNU/Linux
 
-The following Linux distribution versions have been tested and are known to be working.
+The following Linux distributions have been tested and are known to be working with
+KiCad version 5.1.
```

Comment:
Maybe mention "latest stable version" instead of the specific number?

Reply:
I was trying to be specific, because I don't know what people consider the 5.1 nightly builds to be. I was afraid that if we said "stable release", then people would think this didn't apply to the 5.1 nightly builds when in fact it does (since sometimes people do refer to those as testing builds).